data/json/monsters/mammal.json

```diff
@@ -2588,12 +2588,45 @@
     "vision_night": 5,
     "harvest": "mammal_fur",
     "dissect": "dissect_lupine_sample_single",
+    "reproduction": { "baby_monster": "mon_wolf_pup", "baby_count": 6, "baby_timer": 365 },
+    "//": "Wolves breed once a year, 4 to 6 puppies in one litter.",
+    "baby_flags": [ "SPRING" ],
     "path_settings": { "max_dist": 10 },
-    "anger_triggers": [ "STALK", "FRIEND_ATTACKED", "FRIEND_DIED", "PLAYER_WEAK", "PLAYER_CLOSE" ],
+    "anger_triggers": [ "STALK", "FRIEND_ATTACKED", "FRIEND_DIED", "PLAYER_WEAK", "PLAYER_CLOSE", "PLAYER_NEAR_BABY" ],
     "placate_triggers": [ "MEAT" ],
     "zombify_into": "mon_zolf",
     "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "KEENNOSE" ]
   },
+  {
+    "id": "mon_wolf_pup",
+    "type": "MONSTER",
+    "copy-from": "mon_wolf",
+    "name": { "str": "wolf puppy", "str_pl": "wolf puppies" },
+    "description": "An adorable, defenseless wolf pup that will, in time, grow into a not-so-defenseless and debatably-adorable wolf. Unlike a fully-grown wolf, you could tame it, if its parents don't maul you.",
+    "looks_like": "mon_dog_auscattle_pup",
+    "volume": "900 ml",
+    "weight": "1200 g",
+    "hp": 10,
+    "speed": 98,
+    "aggression": -25,
+    "morale": -10,
+    "melee_skill": 2,
+    "melee_dice": 1,
+    "melee_dice_sides": 1,
+    "melee_cut": 1,
+    "dodge": 1,
+    "vision_night": 4,
+    "harvest": "mammal_tiny",
+    "dissect": "dissect_lupine_sample_single",
+    "upgrades": { "age_grow": 42, "into": "mon_wolf" },
+    "petfood": {
+      "food": [ "DOGFOOD" ],
```

Comment:
AFAIK, the policy on taming is only formerly domesticated animals.

Reply:
Ahhhh, fair. I'll remove it. Realistically, you'd, need to condition the puppy, and, we don't have, the, infrastructure for that AFAIK?

Reply:
Removed.

Reply:
Yeah, our taming system is very simplistic is the motivation behind this.

Reply:
> Yeah, our taming system is very simplistic is the motivation behind this.

I hope it can, be, expanded at one point then. I want a pet wolf.

Reply:
sadness... i want a pet wolf as well

Reply:
you're welcome to mod that flag back in if you want.

Reply:
Be sad no more! #57034 Wolf puppies are tamable again with a mod.